src/tcgwars/logic/impl/gen3/PopSeries4.groovy

```diff
@@ -319,23 +313,31 @@ public enum PopSeries4 implements LogicCardInfo {
         pokeBody "Shell Retreat", {
           text "As long as Squirtle has any Energy cards attached to it, damage done to Squirtle by an opponent's attack is reduced by 10 (after applying Weakness and Resistance)."
           delayedA {
+            after APPLY_ATTACK_DAMAGES, {
+              bg.dm().each {
+                if (it.to == self && self.cards.energyCount(C) && it.dmg.value && it.notNoEffect) {
+                  bc "Shell Retreat -10"
+                  it.dmg -= hp(10)
+                }
+              }
+            }
           }
         }
         move "Tackle", {
-          text "10 damage. "
+          text "10 damage."
           energyCost C
           attackRequirement {}
           onAttack {
             damage 10
           }
         }
       };
-      case TREECKO_DELTA_SPECIES__15:
+      case TREECKO_DELTA_15:
       return basic (this, hp:HP040, type:P, retreatCost:1) {
```

Comment:
reprint from Crystal Guardians

`return copy (CrystalGuardians.TREECKO_DELTA_68, this)`